conf/app.conf

```
@@ -4,7 +4,7 @@ runmode = ${RUN_MODE}
 autorender = false
 copyrequestbody = true
 EnableDocs = true
-tipomovimiento = 2
+tipomovimiento = 39
```

Para que el valor adecuado venga por una variable de entorno, y en caso que no esté definida, tome el valor especificado, por defecto
```suggestion
tipomovimiento = ${ID_MOVIMIENTO_PRESUPUESTAL||39}
```